modules/driver-jdbc/src/main/kotlin/datamaintain/db/driver/jdbc/JdbcDriverConfig.kt

```diff
@@ -8,13 +8,14 @@ import mu.KotlinLogging
 import java.util.*
 
 private val logger = KotlinLogging.logger {}
+private const val DB_TYPE = "jdbc"
```

Comment:
Isn't there an enum for db types? Maybe in the CLI. If it's there, it would be nice to move it to the core module and use it here, instead of a constant